brainframe_qt/ui/main_window/video_thumbnail_view/widgets/video_small/video_small.py

```diff
@@ -0,0 +1,98 @@
+from PyQt5.QtCore import QRectF, Qt, pyqtSignal
+from PyQt5.QtGui import QColor, QFontMetricsF, QImage, QPainter, QMouseEvent
+from PyQt5.QtWidgets import QWidget
+from brainframe.api.bf_codecs import StreamConfiguration
+
+from brainframe_qt.api_utils import get_stream_manager
+from brainframe_qt.ui.resources.video_items.streams import StreamWidget
+
+
+class VideoSmall(StreamWidget):
```

Comment:
Do you think a better name would be ThumbnailVideo?

Reply:
yes, but that's a change for a future PR